src/background/Wallet/WalletRecord.ts

```diff
@@ -857,6 +857,7 @@ export class WalletRecordModel {
       enableTestnets: false,
       testnetMode: null,
       enableHoldToSignButton: null,
+      transactionsOnAssetPageChartHidden: false,
```

Comment:
sounds weird and backwards

More commonly this is names something like `showTrades` or `showTradesOnChart` with a default of `true`